resources/FocusPoint.php

```diff
@@ -24,7 +24,7 @@ private function addHooks()
      */
     public function loadScripts()
     {
-        wp_enqueue_script('focuspoint-js', IMAGEFOCUS_ASSETS . 'js/focuspoint.min.js', ['jquery']);
+        wp_enqueue_script('focuspoint-js', IMAGEFOCUS_ASSETS . 'js/focuspoint/focuspoint.js', ['jquery']);
```

Comment:
Got to find a way to make this more easy for testing. Will change it.

I have updated this in the following feature: feature/improve-code-and-functionality